src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -468,6 +481,75 @@ private Locomotive performSwipe(SwipeElementDirection direction, WebElement elem
         driver.swipe(from.getX(), from.getY(), to.getX(), to.getY(), SWIPE_DURATION_MILLIS);
         return this;
     }
+  
+    private Locomotive performCornerSwipe(ScreenCorner corner, SwipeElementDirection direction, float percentage, int duration) {
```

Comment:
the method name is `performCenterSwipe` but it is using a point in the four corners and the swiping in a single direction. The center of the screen is not used in this method. It's more a from corner along edge swipe. What would you think about renaming this method?

Reply:
^ the method is called `performCornerSwipe` ;)

Reply:
My bad, disregard that.